aspnetcore/publishing/index.md

```diff
@@ -12,22 +12,40 @@ ms.technology: aspnet
-
-* [Host on Windows with IIS](xref:publishing/iis)
-  * [ASP.NET Core Module Configuration Reference](xref:hosting/aspnet-core-module)
-  * [IIS Modules with ASP.NET Core](xref:hosting/iis-modules)
-* [Host in a Windows service](xref:hosting/windows-service)
-* [Host on Linux with Nginx](xref:publishing/linuxproduction)
-* [Host on Linux with Apache](xref:publishing/apache-proxy)
-* [Host in Docker](xref:publishing/docker)
-  * [Building Docker Images](https://docs.microsoft.com/dotnet/articles/core/docker/building-net-docker-images)
-  * [Visual Studio Tools for Docker](https://docs.microsoft.com/dotnet/articles/core/docker/visual-studio-tools-for-docker)
-  * [Publish to a Docker Image](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker)
-* [Publish to Azure with Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs)
-* [Continuous deployment to Azure with Visual Studio and Git](xref:publishing/azure-continuous-deployment)
-* [Continuous deployment to Azure with VSTS](xref:publishing/vsts-continuous-deployment)
-* [Publish profiles in Visual Studio](xref:publishing/web-publishing-vs)
-* [Directory structure](xref:hosting/directory-structure)
+# Hosting and deployment overview for ASP.NET Core apps
+
+Here are the main steps you perform to deploy an ASP.NET Core app to a hosting environment:
+
+* Publish the app's runtime files to a folder, and copy the folder to a server.
+* Set up a process manager that starts the app when requests come in and restarts it after it crashes or the server reboots.
+* Set up a reverse proxy that forwards requests to the app.
+
+## Publish runtime files to a folder 
+
+The [dotnet publish](https://docs.microsoft.com/dotnet/articles/core/tools/dotnet-publish) CLI command compiles application code and copies the application files and dependency files into a *publish* folder. When you deploy from Visual Studio the `dotnet publish` step is done for you automatically before deployment.
+
+### Publish folder contents
+
+The *publish* folder contains *.exe* and *.dll* files for the application, its dependencies, and optionally the .NET runtime.
+
+A .NET Core app can be published as *self-contained* or *framework-dependent*. If the app is self-contained, the *.dll* files that contain the .NET runtime are included in the *publish* folder.  If the app is framework-dependent, the .NET runtime files are not included because the app has a reference to a version of .NET that is installed on the computer. The default deployment model is framework-dependent. For more information, see [.NET Core application deployment](https://docs.microsoft.com/dotnet/articles/core/deploying/index).
+
+In addition to *.exe* and *.dll* files, the *publish* folder for an ASP.NET Core app can contain configuration files, static assets, log files, and MVC views.  For more information, see [Directory structure](xref:hosting/directory-structure).
```

Comment:
or maybe typically contains static assets such as ...
I don't think you publish log files - but maybe set up a log folder.

BTW, vishal told me about a neat trick where you can set delete everything on deploy but make an exception for log folders. He just outlined it. That would be a cool feature to blog about.